app/src/main/res/layout/dialog_image_description.xml

```diff
@@ -11,28 +12,53 @@
         android:layout_width="match_parent"
         android:layout_height="0dp"
```

Comment:
Can layout height be set to match_parent here so you don't have to set it in whatever you are doing in the onStart block?

Reply:
This is the image view (with layout_weight=1).

The things in the onStart influence the window/dialog. Normally an (alert) dialog implicitly has something like "wrap_content" that is overwritten there.